### ISSUE TITLE
How can I generate my own training datasets

### ISSUE THREAD

**Issue by SuGoe-Young** [Championchess/A-Tri-Layer-Plugin-to-Improve-Occluded-Detection#6]
I saw the front and back masks can be download，but how can I generate my own training datasets??

**Comment by Championchess** [comment]
You can generate your own training datasets using the methods described in Section 4 of the paper. The idea is 'amodal completion' + 'depth verification'.

**Comment by SuGoe-Young** [comment]
got it thx. another quetions is can I only use amodal completion to generate?
Because I think the way amodal finds the intersection is good enough.

**Comment by Championchess** [comment]
Yes, you can try. But in our project, if we use both the quality of generated training dataset is better.

**Comment by SuGoe-Young** [comment]
Excuse me, would you like to share your source code for generating the training datasets?

**Comment by ShirleySyh** [comment]
> got it thx. another quetions is can I only use amodal completion to generate? Because I think the way amodal finds the intersection is good enough.

Hi, could you share your amodal completion codes? I have been generating the amodal masks of my own dataset for a while, but I'm afraid that I couldn't generate them correctly. Thank you!

**Comment by Championchess** [comment]
You can refer to the official github of De-Occlusion (CVPR 2020 oral) for the amodal completion code and follow the strategies in Section 4 of our paper to generate your own amodal masks.

**Comment by ShirleySyh** [comment]
> You can refer to the official github of De-Occlusion (CVPR 2020 oral) for the amodal completion code and follow the strategies in Section 4 of our paper to generate your own amodal masks.

Thank you. I will try the deocclusion code.
